composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "MIT",
     "require": {
         "neos/form-builder": "^2.3",
-        "hubspot/api-client": "^9.2"
+        "hubspot/api-client": "^9.2",
+        "ext-json": "*"
```

Comment:
Is ext json already required? I dont see any code in the pr? If so, thanks for noticing!

Reply:
I added `ext-json` because of `json_encode` in the helper. It wasn't required before.